cmd/network-passt-binding/main.go

```diff
@@ -43,6 +43,8 @@ const (
 
 func main() {
 	logFile := flag.String("log-file", passtLogFilePathDefault, "file path for Passt logs")
+	queueCount := flag.Uint("queue-count", 0, "queues count to set for the guest interface")
```

Comment:
Is this information available at the virt-controller, when the sidecar manifest is created?
Per what I know, we calculated this late in the virt-handler.

If this is indeed possible in the virt-controller, please elaborate why in the commit message.